### ISSUE TITLE
Incomplete responses 

### ISSUE THREAD

**Issue by VelaJafet** [EvilFreelancer/routeros-api-php#59]
**Version of RouterOS**
I'm  using RB2011UiAS with 7.1beta5 (development) version.

**To Reproduce**

        $client= new Client([
            'host' => $access->DireccionIP,
            'user' => $access->Usuario,
            'pass' => $access->Password,
            'port' => $access->Puerto,
            'ssh_port' => 8722
        ]);

        $query = (new Query('/ip/hotspot/user/profile/print'));

        $response = $client->query($query)->read();

        return response()->json([
            'response' => $response,
            'mensaje' => 'Información de Acceso encontrada',
            'exito' => true
        ]);

**Expected behavior**
I have a PHP function to print all the User Profiles into Ip/Hotspot called with ReactJS as API but the response is always incomplete because I have 23 items (checked it with Winbox) and I receive sometimes 7 objects or 9 objects but never all the objects.

![responsewinbox](https://user-images.githubusercontent.com/43042942/115612853-55f59100-a2b1-11eb-85b0-442ca7db7ce0.PNG)

![responsejson](https://user-images.githubusercontent.com/43042942/115612887-5f7ef900-a2b1-11eb-8336-91aaf7487577.PNG)

**What could be?**
Another question that I don't understand: why do I have to use /ip/hotspot/user/**profile**/print instead /ip/hotspot/**user-profile**/print? Does this relation with "No such command prefix" error?

**Comment by EvilFreelancer** [comment]
Hello @VelaJafet ! Thanks for your report.

> and I receive sometimes 7 objects or 9 objects but never all the objects.

It's strange, probably some issues with modern version of RouterOS, did you tried to reproduce on stable version?

> Another question that I don't understand: why do I have to use /ip/hotspot/user/profile/print instead /ip/hotspot/user-profile/print? Does this relation with "No such command prefix" error?

Probably in new version name of this command changed and you need to use `/ip/hotspot/user-profile/print` instead of `/ip/hotspot/user/profile/print `.

I'll add additional unit tests for 7.x soon

**Comment by VelaJafet** [comment]
I tried with the last stable version of RouterOS (6.48.2) but I have the same issue of incomplete responses :/

**Comment by EvilFreelancer** [comment]
Okay, thanks, then issue in library, i'll figure it out.

**Comment by EvilFreelancer** [comment]
Hello @VelaJafet! Apologize for such a long answer, I tried to reproduce your problem, but always get exactly the number of user profiles that was created. Through tests, I've added 1000 profiles and if make a request to the router, then they all come.

The only thing that I managed to find out is that if I make a request to receive immediately after adding user profiles (without reconnecting the socket session), then the response is empty.
